trackbasket_be/resources/volunteer.py

```diff
@@ -5,13 +5,12 @@
 class Volunteers(Resource):
 
   def get(self,id):
-    volunteer = db.session.query(Volunteer).get(id)
-    return {'volunteer': volunteer.name}
+    volunteer = db.session.query(Volunteer).filter_by(volunteer_id=id).first()
```

Comment:
selects in the table the `volunteer` matching the `id` passed in

Reply:
could we add this query to our model since it's directly interacting with the database?